ca/ca.go

```diff
@@ -427,13 +432,60 @@ func (ca *CertificateAuthorityImpl) IssuePrecertificate(ctx context.Context, iss
 		return nil, err
 	}
 
-	precertDER, err := ca.issuePrecertificateInner(ctx, issueReq, serialBigInt, validity, precertType)
-	if err != nil {
-		return nil, err
+	if features.Enabled(features.PrecertificateOCSP) {
+		serialHex := core.SerialToString(serialBigInt)
+		nowNanos := ca.clk.Now().UnixNano()
+		expiresNanos := validity.NotAfter.UnixNano()
+		_, err = ca.sa.AddSerial(ctx, &sapb.AddSerialRequest{
+			Serial:  &serialHex,
+			RegID:   issueReq.RegistrationID,
+			Created: &nowNanos,
+			Expires: &expiresNanos,
+		})
+		if err != nil {
+			return nil, err
```

Comment:
I think it's reasonable to not audit log in this case but I wanted to make sure you agree and it wasn't an omission. WDYT?

Reply:
Yeah I think that's reasonable. The error will show up in the caller's error logs (i.e. WFE), and nothing has been signed at this point.